### DISCUSSION TITLE
OWC_3 --> pyOWC transition

### DISCUSSION THREAD

**Discussion by dirkslawinski** [euroargodev/argodmqc_owc#103] [Q&A]
HiHo,
  so @kamwal has asked me to give pyOWC a go and see why it falls over for some.  Argo Australia has been using OWC_3 for awhile now ... with some localized adds/fixes.  We did get pyOWC installed-ish when it 1st came out but had not given it a true test.  Now we will.
  
  I note that unlike the Matlab version the "ow_config.txt" info is hardwired in /argodmqc_owc/pyowc/configuration.py.  Was this a conscious decision to template the dictionary in the object which you then update manually or was there scope to read a local file file, say in json, at a later time?

**Comment by gmaze** [comment]
This is not a pythonic design and probably a choice that should change in the future

**Reply by dirkslawinski** [reply]
OK, my local process loads a JSON file to overwrite the templated info.  Just as a suggestion for a future path.

Next step is to sort the numpy version issue.
